### PR TITLE
21233: Remove Scripts Editor option from Edit menu and all related files

### PR DESCRIPTION
**Pull request by mnafees**
Worklist job [21233](https://worklist.net/21233)

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
An admin must verify this patch for the pull request build to fire. Type "ok to test" to begin testing this pull request or "add to whitelist" to test this and all future pull requests from this collaborator.

**Comment by ZappoMan** [comment]
add to whitelist

**Comment by hifi-gustavo** [comment]
Build console output for commit: deacfbf9480f444a2f7673bda74d84b2158c9c49 available at: https://gist.github.com/80b2d7b1a6071cfeb865dadf40746f67<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9945-deacfbf.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9945-deacfbf.dmg

File: libraries/render-utils/src/RenderPipelines.cpp [hunk 1]
```diff
@@ -307,7 +307,7 @@ void initForwardPipelines(render::ShapePlumber& plumber) {
 void addPlumberPipeline(ShapePlumber& plumber,
         const ShapeKey& key, const gpu::ShaderPointer& vertex, const gpu::ShaderPointer& pixel) {
     // These key-values' pipelines are added by this functor in addition to the key passed
-    assert(!key.isWireFrame());
+    assert(!key.isWireframe());
```
Comment (ZappoMan) [inline libraries/render-utils/src/RenderPipelines.cpp]: this is strange, why is this file changed in your PR? The diff doesn't actually look like anything changed. Can you clean this up?

Comment (mnafees) [inline libraries/render-utils/src/RenderPipelines.cpp]: Sure. I had to change this particular line since I was getting compile-time errors.

**Comment by ZappoMan** [comment]
Ok. I now see the difference. That's strange though I wonder if you just need to update from master.

**Comment by mnafees** [comment]
Should I open a separate PR for this?

**Comment by ZappoMan** [comment]
Can you rebase or pull from upstream master? I have to imagine you just got a bad copy of master when you started your branch. If this typo existed in master now, no other PRs would build.

**Comment by mnafees** [comment]
My branch says it is up to date with master and a quick search on GitHub
shows that the typo does exist in the source file.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 396768f787bbb21b51ca560c6b6e344f7210cdac available at: https://gist.github.com/80b2d7b1a6071cfeb865dadf40746f67<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9945-396768f.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9945-396768f.dmg

File: interface/src/ui/ScriptEditorWindow.cpp [hunk 2]
```diff
@@ -1,259 +0,0 @@
...
-
-    connect(new QShortcut(QKeySequence("Ctrl+N"), this), &QShortcut::activated, this, &ScriptEditorWindow::newScriptClicked);
-    connect(new QShortcut(QKeySequence("Ctrl+S"), this), &QShortcut::activated, this,&ScriptEditorWindow::saveScriptClicked);
-    connect(new QShortcut(QKeySequence("Ctrl+O"), this), &QShortcut::activated, this, &ScriptEditorWindow::loadScriptClicked);
-    connect(new QShortcut(QKeySequence("F5"), this), &QShortcut::activated, this, &ScriptEditorWindow::toggleRunScriptClicked);
-
-    _ScriptEditorWindowUI->loadButton->setIcon(QIcon(QPixmap(PathUtils::resourcesPath() + "icons/load-script.svg")));
-    _ScriptEditorWindowUI->newButton->setIcon(QIcon(QPixmap(PathUtils::resourcesPath() + "icons/new-script.svg")));
-    _ScriptEditorWindowUI->saveButton->setIcon(QIcon(QPixmap(PathUtils::resourcesPath() + "icons/save-script.svg")));
-    _ScriptEditorWindowUI->toggleRunButton->setIcon(QIcon(QPixmap(PathUtils::resourcesPath() + "icons/start-script.svg")));
```
Comment (thoys) [inline interface/src/ui/ScriptEditorWindow.cpp]: You might also want to delete these images since they're only used by the editor 👍

**Comment by hifi-gustavo** [comment]
Build console output for commit: c46052398071165c2292f50e8963d52b1ed55990 available at: https://gist.github.com/80b2d7b1a6071cfeb865dadf40746f67<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9945-c460523.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9945-c460523.dmg

**Comment by OmegaHeron** [comment]
Tested Windows 10
QA Pass

No sign of script edit in any menu, keyboard modifier (ALT S) leads to no issue.  Check running scripts, console, and other menu options.  All continue to work as expected.  Smoke test = good.
